src/osd/osd_types.cc

```diff
@@ -335,8 +331,7 @@ void osd_stat_t::encode(bufferlist &bl) const
   ::encode(kb_avail, bl);
   ::encode(snap_trim_queue_len, bl);
   ::encode(num_snap_trimming, bl);
-  ::encode(hb_in, bl);
-  ::encode(hb_out, bl);
+  ::encode(hb_peers, bl);
```

Comment:
we can't drop a field or else we break compatibility.  hb_peers replaces hb_in.  in place of hb_out, ::encode((uint32_t)0, bl);